.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```

Comment:
please remove the .idea files from the PR

Reply:
removed :)